connectors/sources/confluence.py

```diff
@@ -524,11 +554,19 @@ async def get_docs(self, filtering=None):
         Yields:
             dictionary: dictionary containing meta-data of the content.
         """
+        await self.confluence_client.verify_spaces()
+        if self.confluence_client.spaces == ["*"]:
+            cql = "cql=type="
+        else:
+            quoted_spaces = "','".join(self.confluence_client.spaces)
+            cql = f"cql=space in ('{quoted_spaces}') AND type="
```

Comment:
nit: I assume `cql` stands for Confluence Query Language, but what's the actual query here? Is it used to fetch content from configured spaces? I would use a more detailed variable name to make it easier to read.